Assets/Scripts/Augment/AugmentImplementations/Hat/MeshProjectileController.cs

```diff
@@ -24,6 +24,9 @@ public class MeshProjectileController : ProjectileController
     [FormerlySerializedAs("hatSize")] [SerializeField]
     private float size = .2f;
 
+    [SerializeField]
+    private float visualSize = 80f;
```

Comment:
I had the same thought, especially considering the potential discrepancies between the hitbox size and the visual sizes. I just couldn't find a universal conversion rate (for different mesh projectiles) between the VFX units used for scaling and the normal unity scales. SInce these differences are seemingly nonsensical already I decided against premptively scaling the visualSize

Reply:
Okay, case closed then